hugo.toml

```diff
@@ -0,0 +1,69 @@
+baseURL = 'https://tina.pm/'
+languageCode = 'en-us'
+title = "Tina's team website"
+theme = 'introduction'
+
+# For a configuration example of introduction theme, see
+# https://github.com/victoriadrake/hugo-theme-introduction/blob/master/exampleSite/config.toml
+
+DefaultContentLanguage           = "en"                    # Default language for multilingual sites
+
+[params]
+  themeStyle                   = "auto"                  # Choose "light" or "dark" or "auto"
+  # favicon                      = "/img/fav.ico"          # Path to favicon file
+  showMenu                     = true                    # Show navigation menu
+  showRSSButton                = false                   # Show rss button in navigation
+  fadeIn                       = true                    # Turn on/off the fade-in effect
+  fadeInIndex                  = false                   # Turn on/off the fade-in effect on the index page even if fade-in was otherwise turned off
+  dateFormat                   = "Jan 2, 2006"
+  email                        = "info@tina.pm"          # E-mail address for contact section
+
+  # Configure the home page
+  [params.home]
+      introHeight              = "fullheight"            # Input either "medium" or "large" or "fullheight"
+      localTime                = false                   # Show your current local time in contact section
+      timeZone                 = "Europe/Dublin"         # Your timezone as in the TZ* column of this list: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+      timeFormat               = "en-US"                 # Language specific format to use
+  [params.projects]
+      useTwoColumns            = false                   # Use a layout with two columns instead of three
+
+  # Social icons appear in introduction and contact section. Add as many more as you like.
+  # Icon pack "fab" includes social network icons, see: https://fontawesome.com/icons?d=gallery&s=brands&m=free
+  # Icon pack "fas" includes solid style icons, see: https://fontawesome.com/icons?d=gallery&s=solid&m=free
+  [[params.social]]
+      url = "mailto:info@tina.pm"  # For a direct email link, use "mailto:test@example.org".
+      icon = "envelope"         # icon name without the 'fa-'
+      icon_pack = "fas"
+  [[params.social]]
+      url   = "https://mastodon.social/@tina.pm"
```

Comment:
This account does not exist yet, and I am not sure if it is a good idea to have a social media account unless we take the time to post stuff there